src/lib.rs

```diff
@@ -28,6 +28,13 @@
 #[cfg(feature = "std")]
 extern crate std;
 
+#[cfg(feature = "rand")]
+extern crate rand;
+
+#[cfg(feature = "rand")]
+mod random_state;
```

Comment:
Can we add a newline between the modules to make it more obvious that only the former is cfged?